docs/source/services/timestamp_service/overview.rst

```diff
@@ -0,0 +1,51 @@
+========
+Overview
+========
+
+The Timestamp Service exposes an API that allows for clients to request fresh timestamps, which are required by the
+AtlasDB transaction protocol. These timestamps are *logical timestamps* (that is, they have no relationship to actual
+wall-clock time), and are returned as signed 64-bit integers.
+
+Guarantees
+==========
+
+We guarantee that a request for a fresh timestamp will return a strictly greater timestamp than any other timestamp
```

Comment:
The guarantee here is not clear to me. You say we return "a strictly greater timestamp than any other timestamp that may be observed before the request was initiated", but the "observed before" part is not completely clear. 

The example you give below also gives a negative case and I would like to see an example of how we maintain a guarantee.